### DISCUSSION TITLE
how to use old network key

### DISCUSSION THREAD

**Discussion by JacekKac** [Koenkk/zigbee2mqtt#9650] [General]
Did upgrade without backup of configuration and now new key is not the same. 
12:45:07: Configuration is not consistent with adapter state/backup!
Network Key: configured=01030507090______, adapter=114baec1f8de8_______
how to use this old key from adapter in configuration?

**Comment by mausch** [comment]
Check out https://www.zigbee2mqtt.io/advanced/zigbee/03_secure_network.html#change-zigbee-network-encryption-key
